client/js/lounge.js

```diff
@@ -465,7 +471,34 @@ $(function() {
 		return false;
 	});
 
+	sidebar.on("click", ".add-channel", (e) => {
+		const id = $(e.target).data("id");
+		const joinForm = $(`#join_channel_${id}`);
```

Comment:
It doesn't matter much but it looks like most (all? That seems unlikely) of our ids/etc. use dashes instead of underscores.
Also, if you don't want to make this a dynamic id, you could use the id of the parent object and a class on this object. Up to you.

Reply:
fixed